integr8ly/installation
# Intly 1388 secure launcher

## Additional Information

Jira: https://issues.jboss.org/browse/INTLY-1388

Updating launcher to use a secure route. 

## Verification Steps
1. Run uninstall
2. Run this branch
3. Update your github oauth `Authorization callback URL` 
4. The secure route should be accessible from the webapp walkthroughs
![webapp-walkthrough](https://user-images.githubusercontent.com/22113654/54985202-4b6c4e80-4fa8-11e9-81d4-4034f3ed010b.png)
and
![rhs](https://user-images.githubusercontent.com/22113654/54988448-5f1ab380-4fae-11e9-92a6-cc343fa912cf.png)

5. The dashboard url linked from the service instances created by the msbroker in the shared namespaces should point to the secure route
![launcher-shared](https://user-images.githubusercontent.com/22113654/54985395-a1d98d00-4fa8-11e9-9a4b-788114b0fdb6.png)
6. Check that the web console custom links point to secure route
![web console custom links](https://user-images.githubusercontent.com/22113654/54988402-45796c00-4fae-11e9-9c10-1ff3bd44c594.png)
